vector/src/main/res/layout/fragment_ftue_combined_login.xml

```diff
@@ -244,6 +244,18 @@
             app:layout_constraintStart_toStartOf="@id/loginGutterStart"
             app:layout_constraintTop_toBottomOf="@id/loginSubmit" />
 
+        <Button
+            android:id="@+id/loginWithQrCode"
+            style="@style/Widget.Vector.Button.Outlined.Login"
+            android:layout_width="0dp"
+            android:layout_height="60dp"
+            android:layout_marginTop="12dp"
+            android:text="@string/login_scan_qr_code"
+            app:drawableLeftCompat="@drawable/ic_qr_code"
+            app:layout_constraintEnd_toEndOf="@id/loginGutterEnd"
+            app:layout_constraintStart_toStartOf="@id/loginGutterStart"
+            app:layout_constraintTop_toBottomOf="@id/ssoButtonsHeader" />
```

Comment:
Maybe hide by default to avoid blinking.

Reply:
Done.